src/stores/businessStore.ts

```diff
@@ -273,21 +283,26 @@ export const useBusinessStore = defineStore('business', {
      * @param context the Vuex context (passed in automatically)
      */
     loadBusinessInfo (): Promise<any> {
-      // need to return a promise because action is called via dispatch
+      // Need to return a promise because the action is called via dispatch.
```

Comment:
Our code style has been both types of comments: phrases (not capitalized, no period), and full sentences (usually multi-line). So this was OK before.